pages/search.js

```diff
@@ -70,57 +72,30 @@ export default function Search({ data: { users, tags, randUsers } }) {
     }
 
     // checks if there is no keyword between 2 commas and removes the second comma and also checks if the input starts with comma and removes it.
-    setInputValue(inputValue
-      .replace(/,(\s*),/g, ",")
-      .replace(/^,/, ""))
+    setInputValue(inputValue.replace(/,(\s*),/g, ",").replace(/^,/, ""));
+
+    async function fetchUsers(value) {
+      try {
+        const res = await fetch(`api/search/${value.replaceAll(",", "/")}`);
```

Comment:
We prepend our URLs with the environment variable `BASE_URL` as recommended by NextJS (unless this has changed and is no longer required?)